core/objects.go

```diff
@@ -219,6 +219,8 @@ type Challenge struct {
 	// Contains information about URLs used or redirected to and IPs resolved and
 	// used
 	ValidationRecord []ValidationRecord `json:"validationRecord,omitempty"`
+	// RFC3339 timestamp. Required by RFC8555 if status is valid
+	Validated *time.Time `json:"validated,omitempty"`
```

Comment:
RFC3339 specifies a format for serializing the time. Here we just have an in-memory `time.Time`. RFC3339 doesn't apply until we write it somewhere, for instance in JSON. I would instead say something like "The time validation for this challenge started."

Note that when you do the WFE part of this you may need to do some tweaks to ensure we actually emit this as RFC3339 in the serialized JSON (it may be the default), and also to cut off the nanoseconds part.

Reply:
> Note that when you do the WFE part of this you may need to do some tweaks to ensure we actually emit this as RFC3339 in the serialized JSON (it may be the default), and also to cut off the nanoseconds part.

Looks like you can use the `RFC3339` constant specified in: https://golang.org/pkg/time/#pkg-constants

Reply:
Good catch. This was a remnant from when that field was a string. I updated with a _mostly_ copied description from RFC8555 which I like better anyway.